### PR TITLE
Add removal of previous versions to PYME installation instructions

### PR DESCRIPTION
**Pull request by zacsimile**
We now recommend removing previous installations of PYME before installing a new version.

### CODE REVIEW

File: docs/Installation/Installation.rst [hunk 1]
```diff
@@ -158,4 +160,16 @@ msvc for python.
...
 
+Removing a PYME install
+=======================
+
+To upgrade PYME instead of starting fresh, enter ``conda update python-microscopy`` in the Anaconda prompt. Otherwise, read on.
+
+To remove an executable installer on Windows, go to **Start Menu > Settings > Apps**, find `python-microscopy` under
+**Apps & Features**, select it and press *Uninstall*. 
 
+For executable installers on Mac, and conda installations on Windows, Mac and Linux, removing the conda envrionment
```
Comment (David-Baddeley) [inline docs/Installation/Installation.rst]: - executable installers on mac don't install into an evironment
- I think our instructions recommend using `PYME` as the env name

Comment (zacsimile) [reply docs/Installation/Installation.rst]: * They install into an environment on mine, just defined by path name
* Current instructions use `pyme` http://python-microscopy.org/doc/Installation/Installation.html

Comment (zacsimile) [reply docs/Installation/Installation.rst]: * I get rid of my packaged install via `conda remove -p /Users/zachcm/python-microscopy -all`

Comment (zacsimile) [reply docs/Installation/Installation.rst]: Do you recommend switching to `rm -rf /Users/zachcm/python-microscopy`? That's the other way I can think of doing it.

Comment (David-Baddeley) [reply docs/Installation/Installation.rst]: `conda remove -p /Users/zachcm/python-microscopy -all` will work if you are running conda from another miniconda/anaconda installation, but not if it's the only conda you have installed.

Comment (zacsimile) [reply docs/Installation/Installation.rst]: Yeah I guess that's the only way I've tried it. So switch to `rm -rf`?

**Pull request merged**